.gitlab-ci.yml

```diff
@@ -1303,12 +1304,12 @@ windows_choco_offline_7_x64:
 
 publish_choco_7_x64:
   rules:
-    - <<: *if_version_7
+    - <<: *if_triggered_on_tag_7
       when: manual
       allow_failure: true
-  stage: image_deploy
+  stage: publish_packages
   tags: ["runner:windows-docker", "windowsversion:1809"]
-  needs: ["windows_choco_online_7_x64"]
+  needs: ["deploy_staging_windows_tags-a7"]
```

Comment:
Hmm.. if we are not running a tagged pipeline I don't think it makes sense to run the `windows_choco_online_7_x64` job, actually, since it's going to be broken. We could:

1. Move the `windows_choco_online_7_x64` job to a new stage after `deploy7`
2. Make it run only `if_triggered_on_tag_7`
3. Make the `publish_choco_7_x64` job need `windows_choco_online_7_x64` again

Wdyt?